src/main.cc

```diff
@@ -710,6 +711,9 @@ main(int argc, char **argv)
 	memtx_tx_manager_init();
 	crypto_init();
 	systemd_init();
+
+	const int enable_overwrite_user_cert_paths = 0;
```

Comment:
"override_cert_paths_env_vars" seems to be more accurate.
You should also consider separate parameters for overriding SSL_CERT_DIR and SSL_CERT_FILE. Or removing such half-baked functionality completely.

Reply:
Variable was renamed.
This flag is used for test purposes to enable overwriting existing SSL env variables, I think there is no need to create separate arguments